assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -14,8 +14,311 @@
+        __(
+          'Keep the title clear and clean, ideally under %1$d words in less than %2$d characters.',
+          'web-stories'
+        ),
+        MIN_STORY_TITLE_LENGTH,
+        MAX_STORY_TITLE_LENGTH
+      ),
+    },
+    MISSING_COVER: {
+      MAIN_TEXT: __('Missing story cover image', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: minimum cover dimension width. 2: minimum cover dimension height. 3: cover dimensions aspect ratio left. 4: cover dimension aspect ratio right. */
+        __(
+          'Used as the cover for the Story and is representative of the story. Should not have the Story title pre-embedded on it or any other burned in text. Should be at least %1$dpx by %2$dpx and maintain a %3$d:%4$d aspect ratio.',
+          'web-stories'
+        ),
+        COVER_DIMENSION_WIDTH_PX,
+        COVER_DIMENSION_HEIGHT_PX,
+        ASPECT_RATIO_LEFT,
+        ASPECT_RATIO_RIGHT
+      ),
+    },
+    COVER_TOO_SMALL: {
+      MAIN_TEXT: __('Story cover image too small', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: minimum cover dimension width. 2: minimum cover dimension height. 3: cover dimensions aspect ratio left. 4: cover dimension aspect ratio right. */
+        __(
+          'Should be at least %1$dpx by %2$dpx and maintain a %3$d:%4$d aspect ratio.',
+          'web-stories'
+        ),
+        COVER_DIMENSION_WIDTH_PX,
+        COVER_DIMENSION_HEIGHT_PX,
+        ASPECT_RATIO_LEFT,
+        ASPECT_RATIO_RIGHT
+      ),
+    },
+    COVER_WRONG_ASPECT_RATIO: {
+      MAIN_TEXT: __('Story cover image wrong aspect ratio ', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: minimum cover dimension width. 2: minimum cover dimension height. 3: cover dimensions aspect ratio left. 4: cover dimension aspect ratio right. */
+        __(
+          'Should be at least %1$dpx by %2$dpx and maintain a %3$d:%4$d aspect ratio.',
+          'web-stories'
+        ),
+        COVER_DIMENSION_WIDTH_PX,
+        COVER_DIMENSION_HEIGHT_PX,
+        ASPECT_RATIO_LEFT,
+        ASPECT_RATIO_RIGHT
+      ),
+    },
+    LOGO_TOO_SMALL: {
+      MAIN_TEXT: __('Publisher logo is too small', 'web-stories'),
+      HELPER_TEXT: sprintf(
+        /* translators: 1: minimum publisher logo dimension 2: publisher logo dimension ratio */
+        __(
+          'Should be at least %1$dpx by %1$dpx and maintain a %2$d:%2$d aspect ratio.',
+          'web-stories'
+        ),
+        PUBLISHER_LOGO_DIMENSION,
+        PUBLISHER_LOGO_RATIO
+      ),
+    },
+    LINK_ATTACHMENT_CONFLICT: {
+      MAIN_TEXT: __('Link conflict with page attachment', 'web-stories'),
+      HELPER_TEXT: __(
+        'Links in the bottom of a page with a Page Attachment are disabled.',
+        'web-stories'
+      ),
+    },
+    MISSING_VIDEO_POSTER: {
+      MAIN_TEXT: __('Video missing poster', 'web-stories'),
+      HELPER_TEXT: __('', 'web-stories'),
```

Comment:
How should a translator translate an empty string? 🙂 

I'd remove this property if there is no helper text.